.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Please create a detailed report by completing the following information
+labels: 
+
+---
+
+**Environment:**
+ - AWS ParallelCluster / CfnCluster version [e.g. aws-parallelcluster-2.0.0]
+ - OS: [e.g. alinux]
+ - Scheduler: [e.g. SGE]
+
+**Bug description and how to reproduce:**
+A clear and concise description of what the bug is and the steps to reproduce the behavior.
+
+**Additional context:**
+Any other context about the problem. E.g.:
+ - `~/.awsparallelcluster/config` file content and pre/post-install scripts
```

Comment:
I would stay generic, like "configuration file with credentials or any other personal data removed and pre/post-install scripts if any"